app/models/shipit/variable_definition.rb

```diff
@@ -1,18 +1,20 @@
 module Shipit
   class VariableDefinition
-    attr_reader :name, :title, :default
+    attr_reader :name, :title, :default, :select
 
     def initialize(attributes)
       @name = attributes.fetch('name')
       @title = attributes['title']
       @default = attributes['default'].to_s
+      @select = attributes['select'] if attributes['select'] && !attributes['select'].empty?
```

Comment:
Make it `nil` if we don't set it. So we can do `if variable.select` in the view.